src/VehicleSetup/JoystickConfig.qml

```diff
@@ -87,6 +87,20 @@ SetupPage {
                         color:                  __barColor
                     }
 
+                    // Deadband
```

Comment:
It's sort of interesting to display this, but I really wonder whether it will just confuse users as to what it is?